plugin-code/src/test/java/com/trivago/cluecumber/json/pojo/EmbeddingTest.java

```diff
@@ -55,4 +57,51 @@ public void isPlainTextTest() {
         embedding.setMimeType("text/plain");
         assertThat(embedding.isPlainText(), is(true));
     }
+    
+    @Test
+    public void getDecodedDataTest() {
+		String originalInput = "This is getDecodeData() Test !!!";
+		String encodeString = new String(Base64.encodeBase64(originalInput.getBytes()));
+		embedding.setData(encodeString);
+		assertThat(embedding.getDecodedData(),is("This is getDecodeData() Test !!!"));
+    }
+    
+    @Test
+    public void getDecodedDataForXMLTest() {
+		String originalInput = "<?xml version=\"1.0\" encoding=\"UTF-8\"?><note><to>Tove</to><from>Jani</from><heading>Reminder</heading><body>Don't forget me this weekend!</body></note>";
+		String encodeString = new String(Base64.encodeBase64(originalInput.getBytes()));
+		embedding.setData(encodeString);
+        embedding.setMimeType("text/xml");
```

Comment:
Please make sure the indentation is consistent with the rest of the code.

Reply:
Done